navigation/ProyectoSegmentacion/scripts/move_base.py

```diff
@@ -0,0 +1,165 @@
+from geometry_msgs.msg import Twist, Vector3,Pose
+import numpy as np
+from tf.transformations import euler_from_quaternion, quaternion_from_euler
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+import actionlib
+from actionlib_msgs.msg import *
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+import rospkg
+import numpy
+import cv2
+import tf 
+from cv2 import imread,flip,rotate
+import PIL.Image
+import os
+import StringIO
+from geometry_msgs.msg import Point32
+from ipa_building_msgs.msg import MapSegmentationGoal
+from ipa_building_msgs.msg import MapSegmentationResult
+from ipa_building_msgs.msg import RoomInformation
+#from ipa_building_msgs.msg import RoomInformation
+from ipa_building_msgs.msg import MapSegmentationResult,MapSegmentationAction
+class MiNodo():
+
+    def __init__(self):
+      self.flag_estado = False
+
+      #############################################################################  
+      ########### INICIALIZACION DEL NODO, SUSCRIPTORES Y PUBLISHERS ########
+      #############################################################################
+      self.rooms_dictionary = {} # diccionario para almacenar las habitaciones
+      rospy.init_node('mi_nodo')
+      #################### CREO UN SUSCRIPTOR A /MAP PARA OBTENER LOS PUNTOS DE ORIGEN DEL MAPA ################
+      
+      rospy.Subscriber("/map", OccupancyGrid, self.callback)
+     
+     ############ CREO UNA VARIABLE CON LA INFO DEL PACKAGE Y TOMO EL PATH A LA IMAGEN DEL MAPA ############
+      rospack = rospkg.RosPack()
+       
+
+      image_filename = rospack.get_path("ipa_room_segmentation") + "/common/files/test_maps/" + "sweet_house_5" + ".pgm"
+      #image_filename = rospack.get_path("ProyectoSegmentacion") +"/maps/" +"house" + ".pgm"
+      
+      ########### LEO LA IMAGEN Y LA FLIPEO PARA QUE COINCIDA CON LA SEGMENTACION ######
+      img = imread(image_filename,0)
+      #img=rotate(img, cv2.ROTATE_180)
+      img = flip(img, 0)# flipeo vertical
+      
+     # print(img)
+      arr = numpy.array(img)
+      flat_arr = arr.ravel()
+      # convert it to a matrix
+      mapa = np.matrix(flat_arr)
+      mapa=img
+
+        #make non-white pixels black
+      for  y  in range( 0,  len(mapa)):
+        for x in range( 0 , len(mapa[0])):
+          if mapa[y, x]<250: ##aca habia un cast de unsigned char
+            mapa[y,x] = 0
+          #else make it white
+          else:
+            mapa[y, x] = 255
+      self.labeling = Image() 
+      self.cv_image = CvBridge() # aca cambie cv_image por cv_bridge
+      #cv_image.encoding = "mono8"
+      #cv_image.image = map
+      #cv_image.cv2_to_imgmsg
+      #cv_image.toImageMsg(labeling)
+      self.cv_image.image = mapa
+      self.cv_image.encoding = "mono8"
+      self.labeling = self.cv_image.cv2_to_imgmsg(self.cv_image.image, self.cv_image.encoding)
+
+      self.move_base=actionlib.SimpleActionClient("/create1/move_base",MoveBaseAction)
```

Comment:
Esto no funcionaría con múltiples robots, por lo cual deberías eliminar el namespace (`create1`) y ponerlo cuando ejecutes el nodo.

```suggestion
      self.move_base=actionlib.SimpleActionClient("move_base",MoveBaseAction)
```